src/clients/api/queries/useGetHypotheticalLiquidityQueries.ts

```diff
@@ -1,21 +1,22 @@
 import { useQueries, QueryObserverOptions } from 'react-query';
+
 import getHypotheticalAccountLiquidity from 'clients/api/queries/getHypotheticalAccountLiquidity';
-import { IGetVTokenBalancesAllOutput } from 'clients/api/queries/getVTokenBalancesAll';
+import { IGetVTokenBalancesAllOutput } from 'clients/api';
 import { useComptroller } from 'hooks/useContract';
 import FunctionKey from 'constants/functionKey';
 import { Asset } from 'types';
 
-type Options = QueryObserverOptions;
+type Options = QueryObserverOptions; // @TODO: add query specific generics
```

Comment:
I looked into this and the types that I could find in the library wanted options to be an array, which we are using an object. At the time I didn't have internet access so I was only referencing the installed types. Hopefully there is a better solution